src/core/server/rendering/rendering_service.tsx

```diff
@@ -307,15 +314,15 @@ export class RenderingService {
    */
   public checkUrlValid = async (url: string, configName?: string): Promise<boolean> => {
     if (url.match(/\.(png|svg|gif|PNG|SVG|GIF)$/) === null) {
-      this.logger.get('branding').warn(configName + ' config is not found or invalid.');
+      this.logger.get('branding').info(configName + ' config is not found or invalid.');
```

Comment:
After talking with Rocky, we think info probably suits better here. Since if user doesn't input anything, we do not want to show it as a warning in the log.